docs/config/node.rst

```diff
@@ -485,6 +485,15 @@ The meaning of the fields of the are as follows:
   | See :ref:`auth_trust`, :ref:`auth_cert` and :ref:`auth_password` for more
   | information about these methods.
 
+.. _auth.host_based.config.${order}.switch_to_plaintext:
+
+**auth.host_based.config.${order}.switch_to_plaintext**
+  | *Default:* ``false``
+  | *Runtime:* ``no``
+
+  | Specifies whether node can switch to plaintext after establishing secured
+  | connection.
```

Comment:
Maybe we should point out that this should really only be used in combination with some IP restrictions